### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aspdye**
As described in #19792. Not finished yet. Feedback is welcome ;)
It is based on the README.md of "ownCloud Mail".

Thanks to @jancborchardt @Gomez @DeepDiver1975 @enoch85 @ChristophWurst @AndyScherzinger @karlitschek @jakobsack @Xenopathic ;)

My plan is to do use these template (when it is finished) for every README.md of the ownCloud Project (Calendar, Contacts,...).

### CODE REVIEW

**Comment by LukasReschke** [comment]
Looks already pretty awesome! :rocket:

**Comment by MorrisJobke** [comment]
Looks good so far :)

**Comment by enoch85** [comment]
Nice!

File: README.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
-Dependencies:
+**[ownCloud](http://ownCloud.org) gives you freedom and control over your own data.
+A personal cloud which runs on your own server.**
 
-[![Dependency Status](https://www.versioneye.com/user/projects/54f4a2384f3108959a000a16/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54f4a2384f3108959a000a16)
+![](http://fs5.directupload.net/images/151016/u6zpkz7l.png)
 
-[![Dependency Status](https://www.versioneye.com/user/projects/54d1f76f3ca0840b190000c0/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54d1f76f3ca0840b190000c0)
+## Why is this so awesome?
+* :file_folder: **Acess your Data** You can store your files, contacts, calendars and more on a server of your choosing.
```
Comment (phil-davis) [inline README.md]: s/Acess/Access/

**Comment by MorrisJobke** [comment]
@aspdye Can you - once you are ready - squash everything please? I can help you with this if you want. Just ping me in IRC ;) (my name there is morrisjobke ;))

**Comment by aspdye** [comment]
@MorrisJobke I will contact you this week via IRC ;)

File: README.md [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
+[![CodeClimate](https://codeclimate.com/github/owncloud/core/badges/gpa.svg)](https://codeclimate.com/github/owncloud/core)
+[![Dependency Status](https://www.versioneye.com/user/projects/54f4a2384f3108959a000a16/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54f4a2384f3108959a000a16)
+[![Dependency Status](https://www.versioneye.com/user/projects/54d1f76f3ca0840b190000c0/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54d1f76f3ca0840b190000c0)
 
-Dependencies:
+**[ownCloud](http://ownCloud.org) gives you freedom and control over your own data.
+A personal cloud which runs on your own server.**
 
-[![Dependency Status](https://www.versioneye.com/user/projects/54f4a2384f3108959a000a16/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54f4a2384f3108959a000a16)
+![](http://fs5.directupload.net/images/151016/u6zpkz7l.png)
```
Comment (nickvergessen) [inline README.md]: Should be hosted on owncloud.org or in the repo?

Comment (aspdye) [inline README.md]: Of course. Thank you for reporting. Forgotten to change this 😲😲

Comment (MorrisJobke) [inline README.md]: Use demo.owncloud.org ;) There is nothing "more official" ;)

Comment (aspdye) [inline README.md]: @MorrisJobke Yes ;) But where should we host the screenshots?

Comment (MorrisJobke) [inline README.md]: Github? Simply drop then in a comment field here and then copy the URL to the ReadMe ;)

Comment (nickvergessen) [inline README.md]: They expire :speak_no_evil:

Comment (nickvergessen) [inline README.md]: @jospoortvliet can we put the screenshot images on .org?

Comment (MorrisJobke) [inline README.md]: > They expire

Really? When? Because I have seen multiple projects that are doing it that way :see_no_evil:

Comment (aspdye) [inline README.md]: Just put the images on owncloud.org ;) Then - we would not have expire problems or something ;)

Or we just ask how the guys of the mail-readme.md ( @jancborchardt @Gomez @DeepDiver1975 @enoch85 @ChristophWurst @AndyScherzinger @karlitschek @jakobsack @Xenopathic and me :grin:) how they have done that :wink:

Comment (jancborchardt) [inline README.md]: Put the screenshots on https://github.com/owncloud/screenshots (or take some from there). Just use the raw link, there's no expiration.

Comment (jancborchardt) [inline README.md]: cc @jospoortvliet cause he maintains that repo.

@aspdye please don't cc so many people when it's just a question. ;) Someone will quickly help you.

Comment (jospoortvliet) [inline README.md]: We can put the screenshots on .org, no problem with that. Note that there are already a bunch of them on .org, find them here:https://github.com/owncloud/owncloud.org/tree/master/assets/img/screenshots/server you can do a PR to that, no problem.

URL's to the images will be a bit odd - looking like this one: https://owncloud.org/wp-content/themes/owncloudorgnew/assets/img/screenshots/server/eight-two/gallery-wee.png

but I'm sure that that is no big deal.

And yes, the other option is to use https://github.com/owncloud/screenshots and use the raw link, there won't be any changes to that as long as github exists ;-) probably easier, too.

**Comment by aspdye** [comment]
So, I have added the screenshot directly from GitHub ;)

Next Step: "Why is it so awesome?" - Section 
@MorrisJobke @LukasReschke @enoch85 @jospoortvliet @jancborchardt @PVince81
Could you review my work (on this section) and give some input for the points?

**Comment by aspdye** [comment]
https://github.com/owncloud/core/wiki/README.md-Guidelines

**Comment by jancborchardt** [comment]
@aspdye wohooo, this looks great! :) Was just showing the current readme to someone today and noticed again it needs improvement. This is really cool! :+1: 

Can you squash your commits into one? Then we can review, merge it in, and make any further modifications in future pull requests and improve step by step. Please also review this @jospoortvliet @owncloud/designers

**Comment by MorrisJobke** [comment]
:+1: from my side

**Comment by LukasReschke** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
merged in via #20834

**Pull request closed without merging**